src/main/java/net/pms/remote/RemoteWeb.java

```diff
@@ -293,8 +296,9 @@ public void handle(HttpExchange t) throws IOException {
 				r.checkThumbnail();
 				in = r.getThumbnailInputStream();
 			}
+			in = ImagesUtil.convertImage(in, ImageFormat.JPEG);
 			Headers hdr = t.getResponseHeaders();
-			hdr.add("Content-Type", r.getThumbnailContentType());
+			hdr.add("Content-Type", HTTPResource.JPEG_TYPEMIME);
```

Comment:
I'd rather keep the thumbnail format the same as the original file as long as the renderer supports it. PNG being lossless will have some overhead as compared to JPG.